pkgs/applications/editors/inventic/skipper.nix

```diff
@@ -0,0 +1,60 @@
+{ stdenv, lib, fetchzip, autoPatchelfHook, makeWrapper, unzip,
+boost172, curl, libX11, libpng12, libxml2, libxslt, openssl, postgresql, qt5
+, zlib }:
+
+let
+  desktopFileTemplate = ./skipper.desktop;
```

Comment:
We have a function called makeDesktopItem which should be used instead of this extra file.